### PR TITLE
Error overlay should not be dismissable for a server error

### LINKED ISSUE #13955: The error overlay should not be dismissable for a server error
# Bug report

When a "Server Error" (runtime) occurs, the error overlay should not be dismissable. Similar to a compile error, but not full screen.

## To Reproduce

Cause a runtime error in your rendering function on the server.

## Expected behavior

The overlay should not be dismissable. It currently can be minimized and reveals a blank page.

### PR DESCRIPTION
**Pull request by khattakdev**
Fixes: #13955 

For Runtime Errors, Removed Close Functionality to avoid closing Error Dialog.

### CODE REVIEW

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| buildDuration | 12.6s | 12.3s | -261ms |
| nodeModulesSize | 67 MB | 67 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.07 | 2.074 | 0 |
| / avg req/sec | 1207.81 | 1205.46 | -2.35 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.388 | 1.343 | -0.04 |
| /error-in-render avg req/sec | 1801.34 | 1862.16 | ⚠️ +60.82 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..75a9.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 959 B | 959 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| buildDuration | 12.9s | 12.9s | -35ms |
| nodeModulesSize | 67 MB | 67 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..75a9.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 875 kB | 875 kB | ✓ |
| link.js | 914 kB | 914 kB | ✓ |
| routerDirect.js | 906 kB | 906 kB | ✓ |
| withRouter.js | 906 kB | 906 kB | ✓ |
| Overall change | 4.48 MB | 4.48 MB | ✓ |

</details>

</details>
Commit: c4b61aa2ac42c0871277d0107ce0fd2f50096bef

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.1s | -114ms |
| nodeModulesSize | 67 MB | 67 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.192 | 2.245 | ⚠️ +0.05 |
| / avg req/sec | 1140.27 | 1113.74 | -26.53 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.443 | 1.479 | ⚠️ +0.04 |
| /error-in-render avg req/sec | 1732.25 | 1690.19 | -42.06 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..75a9.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 959 B | 959 B | ✓ |
| withRouter.html gzip | 947 B | 947 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| buildDuration | 13.5s | 13.3s | -215ms |
| nodeModulesSize | 67 MB | 67 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.51 kB | 6.51 kB | ✓ |
| webpack-HASH.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..75a9.js gzip | 10.5 kB | 10.5 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 56.9 kB | 56.9 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.6 kB | 5.6 kB | ✓ |
| webpack-HASH..dule.js gzip | 746 B | 746 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 6.92 kB | 6.92 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.4 kB | 52.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.3 kB | 26.3 kB | ✓ |
| Overall change | 26.3 kB | 26.3 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 267 B | 267 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 539 B | 539 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _error.js | 875 kB | 875 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 875 kB | 875 kB | ✓ |
| link.js | 914 kB | 914 kB | ✓ |
| routerDirect.js | 906 kB | 906 kB | ✓ |
| withRouter.js | 906 kB | 906 kB | ✓ |
| Overall change | 4.48 MB | 4.48 MB | ✓ |

</details>

</details>
Commit: 48c6748d54ddb6fbc2ed22e96c97b8ad051cbdc0

File: packages/react-dev-overlay/src/internal/container/Errors.tsx [hunk 1]
```diff
@@ -161,23 +159,10 @@ export const Errors: React.FC<ErrorsProps> = function Errors({ errors }) {
   React.useEffect(() => {
     if (errors.length < 1) {
       setLookups({})
-      setMinimized(false)
       setActiveIndex(0)
     }
   }, [errors.length])
 
-  const minimize = React.useCallback((e?: MouseEvent | TouchEvent) => {
```
Comment (timneutkens) [inline packages/react-dev-overlay/src/internal/container/Errors.tsx:169]: I don't think this should be removed 🤔

Comment (eKhattak) [reply packages/react-dev-overlay/src/internal/container/Errors.tsx:169]: @timneutkens This is never used
code

**Review by Timer — CHANGES_REQUESTED** [review]
Only server-side errors shouldn't be minimizable. Runtime errors still must be minimizable!

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| buildDuration | 12.5s | 12.1s | -449ms |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.017 | 2.097 | ⚠️ +0.08 |
| / avg req/sec | 1239.42 | 1192.15 | ⚠️ -47.27 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.281 | 1.247 | -0.03 |
| /error-in-render avg req/sec | 1951.29 | 2005.5 | +54.21 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..c646.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| index.html gzip | 954 B | 954 B | ✓ |
| link.html gzip | 960 B | 960 B | ✓ |
| withRouter.html gzip | 946 B | 946 B | ✓ |
| Overall change | 2.86 kB | 2.86 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong></summary>

<br/>

<details>
<summary><strong>General</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| buildDuration | 13.2s | 12.9s | -324ms |
| nodeModulesSize | 66.1 MB | 66.1 MB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.js gzip | 6.63 kB | 6.63 kB | ✓ |
| webpack-HASH.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..c646.js gzip | 10.7 kB | 10.7 kB | ✓ |
| framework.HASH.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 57.2 kB | 57.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| main-HASH.module.js gzip | 5.71 kB | 5.71 kB | ✓ |
| webpack-HASH..dule.js gzip | 751 B | 751 B | ✓ |
| 19b7e98f51cc..dule.js gzip | 7.1 kB | 7.1 kB | ✓ |
| framework.HA..dule.js gzip | 39.1 kB | 39.1 kB | ✓ |
| Overall change | 52.7 kB | 52.7 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| polyfills-HASH.js gzip | 26.4 kB | 26.4 kB | ✓ |
| Overall change | 26.4 kB | 26.4 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _buildManifest.js gzip | 268 B | 268 B | ✓ |
| _buildManife..dule.js gzip | 272 B | 272 B | ✓ |
| Overall change | 540 B | 540 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | MLH-Fellowship/next.js arsalan/issue-13955 | Change |
| - | - | - | - |
| _error.js | 876 kB | 876 kB | ✓ |
| 404.html | 4.17 kB | 4.17 kB | ✓ |
| hooks.html | 3.79 kB | 3.79 kB | ✓ |
| index.js | 877 kB | 877 kB | ✓ |
| link.js | 915 kB | 915 kB | ✓ |
| routerDirect.js | 909 kB | 909 kB | ✓ |
| withRouter.js | 909 kB | 909 kB | ✓ |
| Overall change | 4.49 MB | 4.49 MB | ✓ |

</details>

</details>
Commit: c718e0dd182c71375d819feaf2a0a9dec4a7d882

**Review by timneutkens — COMMENTED** [review]
Looks good to me 💯 However I want @Timer to double check 👍

File: packages/react-dev-overlay/src/internal/container/Errors.tsx [hunk 2]
```diff
@@ -223,14 +223,14 @@ export const Errors: React.FC<ErrorsProps> = function Errors({ errors }) {
         type="error"
         aria-labelledby="nextjs__container_errors_label"
         aria-describedby="nextjs__container_errors_desc"
-        onClose={minimize}
+        onClose={!isServerError && minimize}
```
Comment (Timer) [inline packages/react-dev-overlay/src/internal/container/Errors.tsx:226]:
```suggestion
        onClose={isServerError ? undefined : minimize}
```

File: packages/react-dev-overlay/src/internal/container/Errors.tsx [hunk 3]
```diff
@@ -223,14 +223,14 @@ export const Errors: React.FC<ErrorsProps> = function Errors({ errors }) {
...
       >
         <DialogContent>
           <DialogHeader className="nextjs-container-errors-header">
             <LeftRightDialogHeader
               previous={activeIdx > 0 ? previous : null}
               next={activeIdx < readyErrors.length - 1 ? next : null}
-              close={minimize}
+              close={!isServerError && minimize}
```
Comment (Timer) [inline packages/react-dev-overlay/src/internal/container/Errors.tsx:233]:
```suggestion
              close={isServerError ? undefined : minimize}
```

File: packages/react-dev-overlay/src/internal/components/LeftRightDialogHeader/LeftRightDialogHeader.tsx [hunk 4]
```diff
@@ -142,37 +142,39 @@ const LeftRightDialogHeader: React.FC<LeftRightDialogHeaderProps> = function Lef
...
-              d="M6 6L18 18"
-              stroke="currentColor"
-              strokeWidth="2"
-              strokeLinecap="round"
-              strokeLinejoin="round"
-            />
-          </svg>
-        </span>
-      </button>
+      {close && (
```
Comment (Timer) [inline packages/react-dev-overlay/src/internal/components/LeftRightDialogHeader/LeftRightDialogHeader.tsx:145]:
```suggestion
      {close ? (
```

File: packages/react-dev-overlay/src/internal/components/LeftRightDialogHeader/LeftRightDialogHeader.tsx [hunk 5]
```diff
@@ -142,37 +142,39 @@ const LeftRightDialogHeader: React.FC<LeftRightDialogHeaderProps> = function Lef
...
+                d="M6 6L18 18"
+                stroke="currentColor"
+                strokeWidth="2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              />
+            </svg>
+          </span>
+        </button>
+      )}
```
Comment (Timer) [inline packages/react-dev-overlay/src/internal/components/LeftRightDialogHeader/LeftRightDialogHeader.tsx:177]:
```suggestion
      ) : null}
```

**Review by Timer — CHANGES_REQUESTED** [review]

**Comment by Timer** [comment]
Merging since we can't edit this PR. Will fix in a follow-up!

**Pull request merged**